lib/services/coinbase_service/coinbase_service.dart

```diff
@@ -115,8 +115,12 @@ class CoinbaseService implements ICoinbaseService {
     required SessionRequestParams request,
   }) async {
     await _checkInstalled();
+    String? cid;
+    if (chainId != null) {
+      cid = chainId.contains(':') ? chainId.split(':').last : chainId;
+    }
     try {
-      final req = Request(actions: [request.toCoinbaseRequest(chainId)]);
+      final req = Request(actions: [request.toCoinbaseRequest(cid)]);
```

Comment:
does everything work if `cid` is undefined?

Reply:
Good point, I'll check tomorrow.

Reply:
Fixed it!